images/installPlatform.sh

```diff
@@ -51,18 +52,22 @@ if [ -z "$sdkStorageAccountUrl" ]; then
 fi
 if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     set +x
-    isSasTokenEmpty=1 
-    sasToken=$ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN
+
+    if [ "$IS_FROM_DYNAMIC_INSTALLATION" = "true" ]; then
+      # retrieves the token coming from the Dynamic Installation Process.
+      sasToken=$ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN
+    else
+      # retrieves the token coming from the Dockerfiles.
+      sasToken="$(cat $ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN_PATH)"
+    fi
+
     if [ -z "$sasToken" ]; then
-      isSasTokenEmpty=0
+      echo "sasToken is empty for $sdkStorageAccountUrl."
+      echo "sasToken cannot be empty. Please get the valid sasToken."
```

Comment:
in line 52 there is the condition `if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]`.

in condition in line 57, it checks if the `sasToken` variable is empty or not. however, this will not be executed if the storage is not the staging one because of the condition in line 52.

Reply:
ok yeah i see now it only runs for the staging storage account. Still think we should possible exit if these echo statements are hit, so they are the last message that we see, rather than continuing to the download and getting non matching SHA error